components/integrations/commands_container/commands_container.tsx

```diff
@@ -22,51 +33,55 @@ const CommandRoute = ({component: Component, extraProps, ...rest}) => ( //eslint
     />
 );
 
-export default class CommandsContainer extends React.PureComponent {
-    static propTypes = {
-
-        /**
-        * The team data needed to pass into child components
-        */
-        team: PropTypes.object,
+type Props = {
 
-        /**
-        * The user data needed to pass into child components
-        */
-        user: PropTypes.object,
+    /**
+     * The team data needed to pass into child components
+     */
+    team?: Team;
 
-        /**
-        * The users collection
-        */
-        users: PropTypes.object,
+    /**
+     * The user data needed to pass into child components
+     */
+    user?: UserProfile;
 
-        /**
-        * Installed slash commands to display
-        */
-        commands: PropTypes.array,
+    /**
+     * The users collection
+     */
+    users?: RelationOneToOne<UserProfile, UserProfile>;
 
-        /**
-        * Object from react-router
-        */
-        match: PropTypes.shape({
-            url: PropTypes.string.isRequired,
-        }).isRequired,
+    /**
+     * Installed slash commands to display
+     */
+    commands: any[];
 
-        actions: PropTypes.shape({
+    /**
+     * Object from react-router
+     */
+    match: {
+        url: string;
+    };
 
-            /**
-            * The function to call to fetch team commands
-            */
-            loadCommandsAndProfilesForTeam: PropTypes.func.isRequired,
-        }).isRequired,
+    actions: {
 
         /**
-        * Whether or not commands are enabled.
-        */
-        enableCommands: PropTypes.bool,
-    }
+         * The function to call to fetch team commands
+         */
+        loadCommandsAndProfilesForTeam: (teamId: string | undefined) => any;
```

Comment:
This can be written as:
```suggestion
        loadCommandsAndProfilesForTeam: (teamId?: string) => any;
```

Also, the return type should probably be `void` since we're not using the return type anyways.

Reply:
If I change the return type to `void`, the `Promise` line (near line 94) errors out.

I have made the other change in the next commit.

Reply:
Ahh right, it should be `Promise<void>` then.

Reply:
This is a dependency on `actions/integration_actions` which hasn't been converted to tsx yet - and so causes other breakages. Leaving as-is

Reply:
Can you leave a comment for now then so that we can migrate it easily in the future?